macleann/enchiridion-client
# adding Cypress for E2E and unit testing down the line

# Simply adding Cypress for testing down the line

- Wanted to trigger a new docker image build in Jenkins, and I plan on using Cypress a little later so I've installed it now

### Bug fixes:
None

## Type of change


# How to Test?

Make sure you've followed all installation and setup instructions in the README.

```
git fetch origin nm-testing
git checkout nm-testing
npm start
```

- [ ] Need to see if this triggers a new pipeline build

# Checklist:
